.github/workflows/cilium-integration-tests.yaml

```diff
@@ -0,0 +1,155 @@
+name: Cilium Integration Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request_target: {}
+  issue_comment:
+    types:
+      - created
+env:
+  KIND_VERSION: v0.18.0
+  CILIUM_REPO_REF: master
+  CILIUM_CLI_TAG: latest
+
+jobs:
+  cilium-connectivity-tests:
+    timeout-minutes: 360
+    name: Cilium Connectivity Tests
+    runs-on: ubuntu-latest-64-cores-256gb
+    steps:
+      - name: Prepare variables for pushing to master
+        if: github.event_name == 'push'
+        run: |
+          echo "PROXY_IMAGE=quay.io/cilium/cilium-envoy" >> $GITHUB_ENV
+          echo "PROXY_TAG=${{ github.sha }}" >> $GITHUB_ENV
+
+      - name: Prepare variables for PR
+        if: github.event_name == 'pull_request'
+        run: |
+          echo "PROXY_IMAGE=quay.io/cilium/cilium-envoy-dev" >> $GITHUB_ENV
+          echo "PROXY_TAG=${{ github.event.pull_request.head.sha }}" >> $GITHUB_ENV
+
+      - name: Prepare variables for issue comment on a PR
+        if: github.event_name == 'issue_comment' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/cilium-integration-test')
```

Comment:
we are kind of familiar with /test. What do you think?

Reply:
good point, changed it to `/test-cilium-integration`

Reply:
hehe i mean just /test :D

Reply:
oh :D then i tend to "no" because we're not triggering every test - which i would expect from a trigger comment `/test`. it's quite explicit that we're executing the cilium integration tests with explicit version of cilium and/or cilium cli.

:thinking: